readme.md

```diff
@@ -82,7 +93,7 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+  this.eat_bamboo = function() {this.num_bamboo_eaten + 1};
 ```
```

Comment:
An instance method for all pandas would make most sense as an addition to the prototype. For example:
```
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```